### ISSUE TITLE
There are too few hairstyles, beards, and character customization in the game

### ISSUE THREAD

**Issue by TheShuEd** [crystallpunk-14/crystall-punk-14#255]
## Description

We need more!
- [x] Hairs (19/30)
- [x] Beards (14/10)
- [ ] Tiefling horns (9/15)
- [ ] Tiefling tails (animated!) (2/5)
- [ ] Elf ears (4/10)
- [ ] Goblins use displacement maps for hairs and beards

When porting hairstyles from the main game, remember that they should be increased in size in all directions, so that they look beautiful on our doll.

**Comment by TheShuEd** [comment]
it's not urgent, it's not a priority.

**Comment by TheShuEd** [comment]
add goblin
![image](https://github.com/crystallpunk-14/crystall-punk-14/assets/96445749/ae14d468-08e4-4a51-b3ba-387606f86ba4)

**Comment by TheShuEd** [comment]
goblins should use displacement maps for hairs

**Comment by TheShuEd** [comment]
https://github.com/space-wizards/space-station-14/pull/35938
need for goblins

**Comment by TheShuEd** [comment]
I don't quite understand what you mean
🤔 
language barrier problems, can you rephrase?

**Comment by mqole** [comment]
no worries, just suggesting some features that you may be interesting in porting into your server, the first link should have a video to show it

**Comment by TheShuEd** [comment]
Seems to me both PR was sended to upstream too?

**Comment by mqole** [comment]
first time im hearing that... scary, but cool, thanks for telling me!

**Comment by mqole** [comment]
first time im hearing that... scary, but cool, thanks for telling me!

**Comment by TheShuEd** [comment]
Такс ладно, у нас уже слишком МНОГО какстомизации.
